test/unit/app/common/state/bookmarksStateTest.js

```diff
@@ -71,6 +71,59 @@ describe('bookmarkState unit test', function () {
       assert.deepEqual(bookmarksState.getBookmarkOrder(stateWithData), stateWithData.getIn(['cache', 'bookmarkOrder']))
     })
   })
+
+  describe('updateFavicon', function () {
```

Comment:
This section mostly copied from https://github.com/brave/browser-laptop/blob/0213d8f30eedf909690c8876fa9f31205bcf0ff5/test/unit/state/siteUtilTest.js#L1068

@NejcZdovc there might be more tests from the old siteUtil that we can pull over (similar to this). For example, `moveSite`

Reply:
I plan to cover all this functions in #10310, it will be unblocked soon